### DISCUSSION TITLE
Cancelling domain transfer

### DISCUSSION THREAD

**Discussion by Lancito01** [vercel/community#6411] [Help] [answered]
### Summary

Hey there. I recently bought a domain on another registrar and wanted to transfer it to Vercel, so I paid the 20$ needed for the annual subscription to transfer it to Vercel. Turns out, since I registered the domain under 60 days ago, the domain will never be actually transferred, so I paid the 60$ for nothing 😞 Any chance I could get a refund, to re-purchase the domain after the 60 days?

Sorry for tagging you @versecafe but you said I could tag you if I had any questions 😃 

### Steps to Reproduce

.

**Comment by versecafe** [comment, marked as answer]
@Lancito01 When it comes to refunds that would have to be something with the Vercel support team itself @amyegan may be able to help.

**Reply by amyegan** [reply]
Hey @Lancito01. If you [submit a **Domain Renewals** case](https://vercel.com/help) to the Customer Success Team, they'll be able to look into your account details and get you a refund or credit as needed. 🙏

**Reply by Lancito01** [reply]
Thank you. Will do 👍

**Reply by Lancito01** [reply]
Update: Namecheap (the registrar where I bought the domain) gave me the refund and cancelled the domain purchase, so I went ahead and re-bought the domain, this time from Vercel itself to avoid any issues 😓 Could I get some basic instructions as to how I can get my project on that domain? Thank you very much ❤️

**Reply by amyegan** [reply]
No worries! Glad you were able to get your refund. You can find steps for buying and assigning a domain here:  https://vercel.com/docs/getting-started-with-vercel/buy-domain

If you already bought the domain, then the last step is to add it in your project's settings. At that point the domain will be connected to your project. 🎉

**Reply by Lancito01** [reply]
So I bought the domain from scratch in Vercel but I didn't get charged. I assume Vercel detected that the transfer went wrong, and maybe refunded me to credit?

Anyways, the website is up-and-running on my chosen domain. Thank you so much for your help 🙌
